src/ext/java/org/opentripplanner/ext/siri/SiriHttpUtils.java

```diff
@@ -47,17 +46,6 @@ public static InputStream postData(
     return entity.getContent();
   }
 
-  /**
-   * Gets a unique ET-Client-Name HTTP header for this instance of OTP.
-   */
-  public static String getUniqueETClientName(String postFix) {
```

Comment:
Not that this is gone, there is nothing Siri-specific in this class anymore and the one remaining method can be moved to the regular HttpUtils, isn't it?

I'm very happy to take over this task after merging this.